oem/smc/updateservice_test.go

```diff
@@ -56,16 +136,122 @@ var updateServiceBody = `{
   "@odata.etag": "\"e9b94401dae9992fef2e71ef30cbcfdc\""
 }`
 
+const smcSSLCertBody = `{
+  "@odata.type": "#SSLCert.v1_0_0.SSLCert",
+  "@odata.id": "/redfish/v1/UpdateService/Oem/Supermicro/SSLCert",
+  "Id": "SSLCert",
+  "Name": "SSLCert",
+  "VaildFrom": "Oct  9 11:15:00 2024 GMT",
+  "GoodTHRU": "Oct  9 11:15:00 2025 GMT",
+  "Actions": {
+    "#SmcSSLCert.Upload": {
+      "target": "/redfish/v1/UpdateService/Oem/Supermicro/SSLCert/Actions/SmcSSLCert.Upload",
+      "UploadKeyWords@Redfish.AllowableValues": [
+        "cert_file",
+        "key_file"
+      ]
+    }
+  },
+  "@odata.etag": "\"e4be24decdd8b293984fb26e1a78e62a\""
+}`
+
+const smcSSLCertUploadResponse = `{
+  "Success": {
+    "code": "Base.v1_10_3.Success",
+    "message": "Successfully Completed Request. See ExtendedInfo for more information.",
+    "@Message.ExtendedInfo": [
+      {
+        "MessageId": "SMC.1.0.OemSslcertUploaded",
+        "Severity": "OK",
+        "Resolution": "No resolution was required.",
+        "Message": "SSL certificate and private key were successfully uploaded.",
+        "MessageArgs": [
+          ""
+        ],
+        "RelatedProperties": [
+          ""
+        ]
+      }
+    ]
+  }
+}`
+
+const sslCertFile = `-----BEGIN CERTIFICATE-----
+MIIDpDCCAoygAwIBAgIUIf
+-----END CERTIFICATE-----`
+
+const sslKeyFile = `-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Will just need to precede this declaration with

```go
//nolint:gosec
const sslKeyFile = ``
```

Reply:
It's updated.